webview.h

```diff
@@ -108,7 +108,7 @@ WEBVIEW_API void webview_bind(webview_t w, const char *name,
                               void *arg);
 
 // Removes a native C callback that was previously set by webview_bind.
-WEBVIEW_API void webview_unbind(webview_t w, const char *name);
+WEBVIEW_API void *webview_unbind(webview_t w, const char *name);
```

Comment:
The callback is automatically freed inside `unbind` since we know what type that is, and I don't believe the `sync` bool is of much use to the user - it was introduced recently for internal use. The arg is `void *` could be anything since it is user-provided, so that is why I'm returning it so it can be conveniently freed. When the sync bind function is used, it is automatically freed since the library created the `arg` under the hood anyway.

Reply:
I think by process of elimination the only thing left that has any value to the caller is the `arg`. That might be true right now but is it significant enough to lock in and dedicate the return value of `webview_unbind` for the `arg`? It is completely optional to pass in something meaningful as the `arg` to `webview_bind` but returning it from `webview_unbind`makes it seem more meaningful to me than it really is.